packages/core/ios/LottieReactNative/ContainerView.swift

```diff
@@ -143,6 +143,20 @@ class ContainerView: RCTView {
         }
     }
     
+    @objc func setSourceDotLottieURI(_ uri: String) {
+        guard let url = URL(string: uri) else {
+            return
+        }
+        
+        _ = LottieAnimationView(
+            dotLottieUrl: url,
+            configuration: lottieConfiguration,
+            completion: { [weak self] view, error in
```

Comment:
I wonder if there is a way to stream line this so that replaceAnimationView is called like the rest of the setSource methods?

Reply:
Discussed in a call.